src/Card/Card.scss

```diff
@@ -252,6 +252,31 @@
   }
 }
 
+.pgn__card-status {
+  box-shadow: none;
+  margin-bottom: 0;
+
+  p {
+    margin-bottom: 0;
+  }
```

Comment:
Related to the other comment, if the consumer is passing any number of `<p>` tags, we'd only want the last `p` element to have no bottom margin; otherwise, there won't be space between any of the paragraphs.

Reply:
I also took an example from alert and set `margin-bottom: 0;` for `&> :last-child`